gym/core.py

```diff
@@ -71,7 +71,11 @@ def step(self, action: ActType) -> Tuple[ObsType, float, bool, dict]:
 
     @abstractmethod
     def reset(
-        self, *, seed: Optional[int] = None, options: Optional[dict] = None
+        self,
+        *,
+        seed: Optional[int] = None,
+        return_info: Optional[bool] = False,
```

Comment:
```suggestion
        return_info: bool = False,
```

`Optional[bool]` means `Union[bool, None]`. Same changes to the remaining type hints.

Reply:
Sounds good, it looks like I missed that one; will fix.